server/db/models/user.js

```diff
@@ -24,6 +24,10 @@ var schema = new mongoose.Schema({
     },
     google: {
         id: String
+    },
+    screenplay: {
```

Comment:
a user can have more than one screenplay, right?  So, I think we need array brackets around the mongoose.Schema.Types.ObjectId